src/modules/downloadChromium.js

```diff
@@ -1,14 +1,19 @@
 import download from 'download-chromium';
-import { getPuppeteerConfig } from 'israeli-bank-scrapers-core';
+import { getPuppeteerConfig } from '@brafdlog/israeli-bank-scrapers-core';
 
 const revision = getPuppeteerConfig().chromiumRevision;
 
+let chromePath;
+
 export default async function downloadChromium(installPath, onProgress) {
-  onProgress({ percent: 0.02, message: 'Step 1: Downloading Chrome...' });
-  // onProgress: track download progress. receives one argument { percent, transferred, total }
-  return download({
-    revision,
-    installPath,
-    onProgress,
-  });
+  if (!chromePath) {
```

Comment:
I understand that this is for download chrome just in the first time. But how has it worked so far? It also downloaded it only for the first time.

Reply:
Thats true, i see now that the library will prevent it from being downloaded twice. Reverting the change